[2024-10-01T00:43:24.309-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T00:43:24.317-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_test.elt_documento_pagar_spark manual__2024-10-01T03:43:21.499319+00:00 [queued]>
[2024-10-01T00:43:24.322-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_test.elt_documento_pagar_spark manual__2024-10-01T03:43:21.499319+00:00 [queued]>
[2024-10-01T00:43:24.323-0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T00:43:24.340-0300] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): elt_documento_pagar_spark> on 2024-10-01 03:43:21.499319+00:00
[2024-10-01T00:43:24.342-0300] {standard_task_runner.py:72} INFO - Started process 8178 to run task
[2024-10-01T00:43:24.346-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_test', 'elt_documento_pagar_spark', 'manual__2024-10-01T03:43:21.499319+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpdcuovuew']
[2024-10-01T00:43:24.348-0300] {standard_task_runner.py:105} INFO - Job 110: Subtask elt_documento_pagar_spark
[2024-10-01T00:43:24.385-0300] {task_command.py:467} INFO - Running <TaskInstance: dag_test.elt_documento_pagar_spark manual__2024-10-01T03:43:21.499319+00:00 [running]> on host DESKTOP-PEJJLJO.
[2024-10-01T00:43:24.443-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_test' AIRFLOW_CTX_TASK_ID='elt_documento_pagar_spark' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T03:43:21.499319+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T03:43:21.499319+00:00'
[2024-10-01T00:43:24.444-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T00:43:24.457-0300] {base.py:84} INFO - Retrieving connection 'spark'
[2024-10-01T00:43:24.459-0300] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark://127.0.0.1:7077 --conf spark.mongodb.input.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.mongodb.output.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.network.timeout=10000000 --conf spark.executor.heartbeatInterval=10000000 --conf spark.storage.blockManagerSlaveTimeoutMs=10000000 --conf spark.driver.maxResultSize=20g --packages org.mongodb.spark:mongo-spark-connector_2.12:3.0.0,com.microsoft.sqlserver:mssql-jdbc:8.4.1.jre8 --total-executor-cores 3 --executor-memory 8g --name arrow-spark --deploy-mode client ./dags/sparkjob.py 2024-10-01 sqlserver://127.0.0.1:1433;databaseName=Teste
[2024-10-01T00:43:25.859-0300] {spark_submit.py:579} INFO - 24/10/01 00:43:25 WARN Utils: Your hostname, DESKTOP-PEJJLJO resolves to a loopback address: 127.0.1.1; using 10.255.255.254 instead (on interface lo)
[2024-10-01T00:43:25.861-0300] {spark_submit.py:579} INFO - 24/10/01 00:43:25 WARN Utils: Set SPARK_LOCAL_IP if you need to bind to another address
[2024-10-01T00:43:26.035-0300] {spark_submit.py:579} INFO - :: loading settings :: url = jar:file:/home/gabzarp/.local/lib/python3.10/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-10-01T00:43:26.123-0300] {spark_submit.py:579} INFO - Ivy Default Cache set to: /home/gabzarp/.ivy2/cache
[2024-10-01T00:43:26.123-0300] {spark_submit.py:579} INFO - The jars for the packages stored in: /home/gabzarp/.ivy2/jars
[2024-10-01T00:43:26.127-0300] {spark_submit.py:579} INFO - org.mongodb.spark#mongo-spark-connector_2.12 added as a dependency
[2024-10-01T00:43:26.127-0300] {spark_submit.py:579} INFO - com.microsoft.sqlserver#mssql-jdbc added as a dependency
[2024-10-01T00:43:26.128-0300] {spark_submit.py:579} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-636c6f22-7327-4c49-87bd-ea44f14adea0;1.0
[2024-10-01T00:43:26.128-0300] {spark_submit.py:579} INFO - confs: [default]
[2024-10-01T00:43:26.217-0300] {spark_submit.py:579} INFO - found org.mongodb.spark#mongo-spark-connector_2.12;3.0.0 in central
[2024-10-01T00:43:26.242-0300] {spark_submit.py:579} INFO - found org.mongodb#mongodb-driver-sync;4.0.5 in central
[2024-10-01T00:43:26.259-0300] {spark_submit.py:579} INFO - found org.mongodb#bson;4.0.5 in central
[2024-10-01T00:43:26.280-0300] {spark_submit.py:579} INFO - found org.mongodb#mongodb-driver-core;4.0.5 in central
[2024-10-01T00:43:26.301-0300] {spark_submit.py:579} INFO - found com.microsoft.sqlserver#mssql-jdbc;8.4.1.jre8 in central
[2024-10-01T00:43:26.317-0300] {spark_submit.py:579} INFO - :: resolution report :: resolve 182ms :: artifacts dl 6ms
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - :: modules in use:
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - com.microsoft.sqlserver#mssql-jdbc;8.4.1.jre8 from central in [default]
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - org.mongodb#bson;4.0.5 from central in [default]
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - org.mongodb#mongodb-driver-core;4.0.5 from central in [default]
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - org.mongodb#mongodb-driver-sync;4.0.5 from central in [default]
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - org.mongodb.spark#mongo-spark-connector_2.12;3.0.0 from central in [default]
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - |                  |            modules            ||   artifacts   |
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - |       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - |      default     |   5   |   0   |   0   |   0   ||   5   |   0   |
[2024-10-01T00:43:26.318-0300] {spark_submit.py:579} INFO - ---------------------------------------------------------------------
[2024-10-01T00:43:26.321-0300] {spark_submit.py:579} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-636c6f22-7327-4c49-87bd-ea44f14adea0
[2024-10-01T00:43:26.322-0300] {spark_submit.py:579} INFO - confs: [default]
[2024-10-01T00:43:26.328-0300] {spark_submit.py:579} INFO - 0 artifacts copied, 5 already retrieved (0kB/6ms)
[2024-10-01T00:43:26.483-0300] {spark_submit.py:579} INFO - 24/10/01 00:43:26 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-01T00:43:26.702-0300] {spark_submit.py:579} INFO - python3: can't open file '/home/gabzarp/airflow/dags/sparkjob.py': [Errno 2] No such file or directory
[2024-10-01T00:43:26.719-0300] {spark_submit.py:579} INFO - 24/10/01 00:43:26 INFO ShutdownHookManager: Shutdown hook called
[2024-10-01T00:43:26.720-0300] {spark_submit.py:579} INFO - 24/10/01 00:43:26 INFO ShutdownHookManager: Deleting directory /tmp/spark-f8638001-ef2f-4e46-a545-310b449d637f
[2024-10-01T00:43:26.764-0300] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://127.0.0.1:7077 --conf spark.mongodb.input.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.mongodb.output.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.network.timeout=10000000 --conf spark.executor.heartbeatInterval=10000000 --conf spark.storage.blockManagerSlaveTimeoutMs=10000000 --conf spark.driver.maxResultSize=20g --packages org.mongodb.spark:mongo-spark-connector_2.12:3.0.0,com.microsoft.sqlserver:mssql-jdbc:8.4.1.jre8 --total-executor-cores 3 --executor-memory 8g --name arrow-spark --deploy-mode client ./dags/sparkjob.py 2024-10-01 sqlserver://127.0.0.1:1433;databaseName=Teste. Error code is: 2.
[2024-10-01T00:43:26.768-0300] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=dag_test, task_id=elt_documento_pagar_spark, run_id=manual__2024-10-01T03:43:21.499319+00:00, execution_date=20241001T034321, start_date=20241001T034324, end_date=20241001T034326
[2024-10-01T00:43:26.784-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T00:43:26.784-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 110 for task elt_documento_pagar_spark (Cannot execute: spark-submit --master spark://127.0.0.1:7077 --conf spark.mongodb.input.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.mongodb.output.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.network.timeout=10000000 --conf spark.executor.heartbeatInterval=10000000 --conf spark.storage.blockManagerSlaveTimeoutMs=10000000 --conf spark.driver.maxResultSize=20g --packages org.mongodb.spark:mongo-spark-connector_2.12:3.0.0,com.microsoft.sqlserver:mssql-jdbc:8.4.1.jre8 --total-executor-cores 3 --executor-memory 8g --name arrow-spark --deploy-mode client ./dags/sparkjob.py 2024-10-01 sqlserver://127.0.0.1:1433;databaseName=Teste. Error code is: 2.; 8178)
Traceback (most recent call last):
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/gabzarp/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://127.0.0.1:7077 --conf spark.mongodb.input.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.mongodb.output.uri=mongodb://127.0.0.1:27017/Financeiro --conf spark.network.timeout=10000000 --conf spark.executor.heartbeatInterval=10000000 --conf spark.storage.blockManagerSlaveTimeoutMs=10000000 --conf spark.driver.maxResultSize=20g --packages org.mongodb.spark:mongo-spark-connector_2.12:3.0.0,com.microsoft.sqlserver:mssql-jdbc:8.4.1.jre8 --total-executor-cores 3 --executor-memory 8g --name arrow-spark --deploy-mode client ./dags/sparkjob.py 2024-10-01 sqlserver://127.0.0.1:1433;databaseName=Teste. Error code is: 2.
[2024-10-01T00:43:26.807-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-01T00:43:26.814-0300] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-01T00:43:26.815-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
